src/ConfigValidator.php

```diff
@@ -111,6 +111,7 @@ private function validateExportConfig(): void
         }
         $this->input = $configData['storage']['input'];
         $this->type = $configData['parameters']['type'];
+        $this->script = implode("\n", $configData['parameters']['script']);
```

Comment:
do we want to validate that `['parameters']['script']` is an array?

Reply:
yes, we do! 

https://github.com/keboola/data-loader/blob/master/src/ExportConfig.php#L30

Reply:
LOL, sorry 🤦‍♂️ 👍